src/gui/group/GroupView.cpp

```diff
@@ -45,6 +45,13 @@ GroupView::GroupView(Database* db, QWidget* parent)
 
     new QShortcut(Qt::CTRL + Qt::Key_F10, this, SLOT(contextMenuShortcutPressed()), nullptr, Qt::WidgetShortcut);
 
+    // keyboard shortcuts to sort children of a group
+    auto shortcut = new QShortcut(Qt::CTRL + Qt::Key_Down, this);
+    connect(shortcut, &QShortcut::activated, this, [this]() { sortGroupRequested(false); });
```

Comment:
Why a new function, just call sortGroups directly right here.

Reply:
Oh..yes.. Correct.
Initially, I made changes as per SIGNAL - SLOT mechanism and with different strategy. I just removed that function now and it's working fine.
Thanks.